### DISCUSSION TITLE
questions regarding to the quality of data

### DISCUSSION THREAD

**Discussion by IllgamhoDuck** [duchallenge-team/dude#3] [Q&A] [answered]
Hi all! Thanks for supporting this awesome competition!

I checked the 2.17 update of data! and found some non-sense questions have been revised (ex: ?, 33?, 1?)
and glad to see that I don't need to create a not-answerable and list dataset.

I have the following questions!

##  Q1:  Won't there be any QA for the documents w/o QA data?
> there were 56 PDFs that doesn't have a QA dataset. and for 2.17 I found it reduced to 35 PDFs. the list is below
```python
  ['3dc85b6ad3e434bb45180cec2116d914', '0fe5e17d2b8120d112a1eb1e215b6be4', '20645e28189c2c9e864319677f472a51', 'f741d7e2ac28340d6eb9dac6b42af870', '0b4b398eaf0e8cec726e54307ecbd8a1', 'efe26ec90a40b899e70266a096fe05de', 'adedc5dfc69216edfccbd4cddb6f0022', 'cad870f8215019e090c5c2df82d6b569', 'a996b1dbc335eb850625abdd76a212fa', '1fc56064e00e1d5325af94487ced56ba', 'cf7517e471dbb1b9e7bc47bcdf56748e', '046bdbb9b5cdff4c6d67cd86d3625c1c', 'f53c70d0446ddec85810357b3259e8d4', 'f7177b882ed10f88572ff8995f3e5f0c', '789ac402b2a87c1dfbe38a0c22c5a509', '021053650d1807059ee5c9def87e58b7', '9911b8d888ca9b557947f314525c4726', '41110179cd47f9e24a12cb851c4839c2', '9ccde808e127eb4492ec8e841a002666', '7acbc5f1f0cf722be93584b0ae272ee8', '4cd9684ac6cbe51141380db56f355e4d', '79469d39e18ccba82306c4afc283b2a5', '01dd2b2f14a8cc91332562b5ff601163', '5fe0057c4c2ab471d263f59b31aaab9d', '2b29039e42d7fef708af57ec8faf8567', '3fc44bd40113e476389accd9975fe9b2', '4d69285acf24fe769a1b430fc056bd0e', '4ddf7fad0a39e0d3105d3413213eb12c', 'c0861808ce62e61adcdf79e66cec2ef7', 'c7f1aec05845e743b31a67bec04a95db', 'f4138ed5e04acab5dc13c7fcdc557248', '5c25f88688acd69f44509f1f1baa312c', '45dd8b8b734577fcb92bb8883d14e380', '0e2585cdde53701a0fbaf1fc6b31a045', '8a25853a013600e52431f6075c2b97f1']
```

will the user need to make their own question to use these images?

## Q2: what is the difference between an abstractive type answer which has more than 2 answers and a list type answer?
At abstractive type answers, there are list-type answers also. they also have more than 1 element for an answer. I really don't feel it is different from the list. Actually, I thought there would be only 2 types of answers at the first sight because of the following phrase on the DUDE page.
> Answers. To cover all possible business use cases, we opted to provide both abstractive and extractive answers. These cover various answer types, such as textual, numerical, dates, yes/no, lists, or non-answerable. The latter demands that the model correctly identifies that the answer cannot be provided, as the question needs to be better formed, e.g., it asks about the value of an empty cell in the table.
- not-answerable makes sense to have its own question type.
- list-type also makes sense. but why does list-type answers in abstractive not assigned as a list?

## Q3: there are some noisy questions. will these be further revised?

- `a75d54111d83706579ec230f6cd6cae9` - it has some broken question
![image](https://user-images.githubusercontent.com/27937120/220414063-6c1cce5d-392a-40eb-9d97-1c1fc1db41ea.png)
- `3a95effab83189eb50781e3d3e5fc8a4` - it asked title but the answer is `yes`
![image](https://user-images.githubusercontent.com/27937120/220414564-f34d2ed8-627e-4841-9984-ecea30842375.png)
- `bf727725e7ebb797186397477179b282` - `not-answerable` replies `no info`. for this case I actually found rarely that not-answerable is replying an answer like "can't answer that question"
![image](https://user-images.githubusercontent.com/27937120/220415013-f330e991-9d84-4099-9904-e4ce26d368d2.png)

and more! just curious will these noisy questions will be further revised with data cleansing or will just remain.

**Comment by Jordy-VL** [comment, marked as answer]
Q1:  substantial effort has gone into collecting and cleaning the current QA pairs for the documents in trainval. After cleaning, some documents inevitably ended up without any validated questions. You are free to create your own QA pairs for these documents or ignore them. 

Q2: after an internal discussion, we decided to standardize the answer type into 5 types: 
`extractive, abstractive, non-answerable, list/extractive, list/abstractive`. 
The current trainval follows this distribution of answer types: 
```
extractive          46.626378
abstractive         35.864657
not-answerable      10.552530
list/extractive      5.498738
list/abstractive     1.457697
```
The latest annotations file has been updated in Zenodo and the HF data loader to reflect this.

Q3: for now we are focusing our data cleaning efforts on the blind test set, so we will not directly treat noise in trainval. Participants are free to handle these noisy data points as they see fit.

**Reply by IllgamhoDuck** [reply]
thanks for the elaborated answer! appreciate it!
